pandas/conftest.py

```diff
@@ -760,6 +760,29 @@ def index_or_series_obj(request):
     return _index_or_series_objs[request.param].copy(deep=True)
 
 
+_object_series = {
```

Comment:
Nit: Could you rename this to `_typ_objects_series`?

Reply:
Done!